### DISCUSSION TITLE
Podman reset and re-install

### DISCUSSION THREAD

**Discussion by rachelglenn** [containers/podman#12910] [Q&A]
I was having issues with Podman and was trying change the uid and gid with podman and user on Ubuntu 18.04. I decided to reset Podman and erase all images, containers, and etc. I ran

podman system reset

sudo apt remove podman

sudo apt -y install podman

I now get the error with any podman commands:
Error: cannot setup namespace using newuidmap: exit status 1

How do I reset everything and reinstall podman from scratch to resolve this issue?

Thanks,

Rachel

**Comment by mheon** [comment]
Can you try rebooting the system? The steps you've described should work, but it looks like something is preventing Podman from creating a new user namespace; it's unclear exactly what, given the unhelpful error from newuidmap.

**Reply by rachelglenn** [reply]
I tried rebooting it and I can get commands to work. However the uid and gid are still messed up. I have read everything to try to fix them. I erased the /etc/uid and gid files. I have tried the troubleshooting suggestions as well. Do you have any resources for setting it up the uid and guid from scratch?

Thanks,

Rachel

**Reply by mheon** [reply]
Messed up how - can you provide details?
